src/components/CMessageRow.vue

```diff
@@ -96,7 +98,8 @@ export default {
       let expectedTime = null;
       if (this.progress && this.progress > 0) {
         const factor = (1 - this.progress) / this.progress;
-        expectedTime = formatDistance(new Date() * factor, this.startedDatetime * factor);
+        const diff = utils.timeUnitsBetween(this.startedDatetime * factor, new Date() * factor);
```

Comment:
what could be better is to add a column `remaining time` with the remaining time.

Also use you new function to display wait time and execution time.